diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 41b4359..930f33e 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -247,9 +247,17 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
 
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
-    extractLibs "com.facebook.fbjni:fbjni:+:headers"
-    extractLibs "com.facebook.fbjni:fbjni:+"
+  // if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  //   extractLibs "com.facebook.fbjni:fbjni:+:headers"
+  //   extractLibs "com.facebook.fbjni:fbjni:+"
+  // }
+
+  def rnVersionParts = REACT_NATIVE_VERSION.tokenize('.')
+  def rnMajor = rnVersionParts.size() > 0 ? rnVersionParts[0].toInteger() : 0
+  def rnMinor = rnVersionParts.size() > 1 ? rnVersionParts[1].toInteger() : 0
+  if (rnMajor == 0 && rnMinor < 71) {
+      extractLibs "com.facebook.fbjni:fbjni:+:headers"
+      extractLibs "com.facebook.fbjni:fbjni:+"
   }
 
   // By default it will just include onnxruntime full aar package
